MAUI/Cartesian-Charts/StackedLine.md

```diff
@@ -0,0 +1,445 @@
+                                  YBindingPath="Value"
+                                  ShowMarkers="True">
+            <chart:StackingLineSeries.MarkerSettings>
+                <chart:ChartMarkerSettings Type="Diamond" 
+                                           StrokeWidth="2"
+                                           Fill="White"
+                                           Height="15"
+                                           Width="10"/>
+            </chart:StackingLineSeries.MarkerSettings>    
+        </chart:StackingLineSeries> 
+
+        <chart:StackingLineSeries ItemsSource="{Binding Data4}"
+                                  XBindingPath="Month"
+                                  YBindingPath="Value"
+                                  ShowMarkers="True">
+            <chart:StackingLineSeries.MarkerSettings>
+                <chart:ChartMarkerSettings Type="Diamond" 
+                                           StrokeWidth="2"
+                                           Fill="White"
+                                           Height="15"
+                                           Width="10"/>
+            </chart:StackingLineSeries.MarkerSettings>    
+        </chart:StackingLineSeries> 
+
+    </chart:SfCartesianChart>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+    SfCartesianChart chart = new SfCartesianChart();
+
+    ViewModel viewModel = new ViewModel();
+    ...
+    ChartMarkerSettings chartMarker= new ChartMarkerSettings();
+        chartMarker.Type = ShapeType.Diamond;
+        chartMarker.Fill=Colors.White;
+        chartMarker.StrokeWidth= 2;
+        chartMarker.Height = 15;
+        chartMarker.Width = 10;
+
+    StackingLineSeries series1 = new  StackingLineSeries()
+    {
+        XBindingPath = "Month",
+        YBindingPath = "Value",
+        ItemsSource = viewModel.Data1,
+        ShowMarkers= true,
+        MarkerSettings = chartMarker,
+    };
+
+    StackingLineSeries series2 = new StackingLineSeries()
+    {
+        XBindingPath = "Month",
+        YBindingPath = "Value",
+        ItemsSource = viewModel.Data2,
+        ShowMarkers= true,
+        MarkerSettings = chartMarker,
+    };
+
+    StackingLineSeries series3 = new StackingLineSeries()       
+    {
+        XBindingPath = "Month",
+        YBindingPath = "Value",
+        ItemsSource = viewModel.Data3,
+        ShowMarkers= true,
+        MarkerSettings = chartMarker,
+    };
+
+    StackingLineSeries series4 = new StackingLineSeries()
+    {
+        XBindingPath = "Month",
+        YBindingPath = "Value",
+        ItemsSource = viewModel.Data4,
+        ShowMarkers= true, 
+        MarkerSettings = chartMarker,
+    };
+
+    chart.Series.Add(series1);
+    chart.Series.Add(series2);     
+    chart.Series.Add(series3); 
+    chart.Series.Add(series4); 
+
+    this.Content = chart;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Stacked Line Custom Marker support in MAUI Chart](Chart-types-images/stacked_line_custom_marker_support.png)
```

Comment:
Remove the image for marker customization

Reply:
Removed image